Sources/edge/Commands/RunCommand.swift

```diff
@@ -58,7 +85,16 @@ struct RunCommand: AsyncParsableCommand {
         logger.info("Loading into Docker")
         try await Shell.run(["docker", "load", "-i", "\(executableTarget.name)-container.tar"])
 
-        logger.info("Running container")
-        try await Shell.run(["docker", "run", "--rm", imageName])
+        if debug {
+            logger.info("Running container with debugger")
+            try await Shell.run([
+                "docker", "run", "--rm", "-it", "-p", "4242:4242",
+                "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined", imageName,
+                "ds2", "gdbserver", "0.0.0.0:4242", "/bin/\(executableTarget.name)",
+            ])
+        } else {
+            logger.info("Running container")
```

Comment:
does this logger do some sort of tags or labels? 
sonething like:

`logger.info("Running container", "{imageNname: $imageName}") ` or something similar

Reply:
Yes, good catch. It's possible to attach metadata to each log message (or logger). I'll add that!

Reply:
edca61e7a2fe9fb1bc1210d86f729c7b7a2c1b69